Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.9
+WORKDIR /app
+COPY requirements.txt /app/
+COPY . /app/
+RUN pip install -r requirements.txt
+EXPOSE 8000
+CMD [ "python3","manage.py","runserver","0.0.0.0:8000" ]
```

Comment:
any particular reason to use CMD command? How do you decide which one you will use when between `CMD vs ENTRYPOINT` ==> add a note for it in README.md